book/python/api.md

```diff
@@ -187,15 +187,15 @@ Partition Keys must correctly support the `__eq__` (`==`) and `__hash__` operato
 
 ### PartitionFunction
 
-A partition function class must be wrapped in the `@partition_function` decorator and return the appropriate [Key](#key) for `data`.
+A partition function class must be wrapped in the `@wallaroo.partition` decorator and return the appropriate [Key](#key) for `data`.
```

Comment:
Not a class anymore!
should be 
`A partition function must be decorated with ...` (also note `decorated with` instead of `wrapped in`)